include/mbedtls/config.h

```diff
@@ -3535,6 +3554,22 @@
  */
 //#define MBEDTLS_SSL_CID_PADDING_GRANULARITY 16
 
+/** \def MBEDTLS_SSL_TLS13_PADDING_GRANULARITY
```

Comment:
The precedent is the class of macros `MBEDTLS_SSL_PROTO_TLS1_X`. I found the current form easier to read, but since @gbryant-arm independently raised the issue and found it confusing, I'll change it.

Reply:
Done.